src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3411,6 +3435,21 @@ namespace SurfaceGeometry {
                                         cCurrentModuleObject + "=\"" + state.dataSurfaceGeometry->SurfaceTmp(SurfNum).Name + "\", " +
                                             state.dataIPShortCut->cAlphaFieldNames(2) + " not found=" + state.dataIPShortCut->cAlphaArgs(2));
                         ErrorsFound = true;
+                    } else {
+                        if (state.dataSysVars->shadingMethod == DataSystemVariables::ShadingMethod::PixelCounting) {
+                            ShowSevereError(state,
+                                            format("{}=\"{}\" has an active {}=\"{}\";",
+                                                   cCurrentModuleObject,
+                                                   state.dataSurfaceGeometry->SurfaceTmp(SurfNum).Name,
+                                                   state.dataIPShortCut->cAlphaFieldNames(2),
+                                                   state.dataIPShortCut->cAlphaArgs(2)));
+                            ShowContinueError(state, "The Shading Calculation Method of choice is \"PixelCounting\". With this choice, ");
+                            ShowContinueError(
+                                state,
+                                "the shading surfaces will be treated as completely opaque (transmittance = 0) during the shading calculation, ");
+                            ShowContinueError(state, "which may result in significant inaccurate or unexpected results.");
+                            ShowContinueError(state, "It is suggested to use another Shading Calculation Method, such as \"PolygonClipping\".");
+                        }
                     }
```

Comment:
This is the adequate location to add the warning, for the `Detached` shading surfaces types.

Reply:
I would move this (well, move setting the global flag) to a separate `if(SchedMaxValue > 0.0) state.dataSolarShading->anyScheduledShadingSurface = true;`
If a surface has a transmittance schedule that is always zero, there's no need to warn, it will be opaque anyway.

Reply:
Thanks! @mjwitte This method is now implemented.

Reply:
Then this else can be removed?